raiden_libs/test/mocks/client.py

```diff
@@ -230,3 +231,11 @@ def sign_data(data, privkey):
         sha3 = lambda x: keccak_256(x).digest()
         sig = pk.sign_recoverable(data, hasher=sha3)
         return sig[:-1] + chr(sig[-1] + 27).encode()
+
+    @assert_channel_existence
+    def get_channel_participant_info(self, partner_address: str) -> Tuple:
```

Comment:
I meant some info about the included data. The contract returns the follwoing:
```
return (
            participant_state.initialized,
            participant_state.deposit,
            participant_state.transferred_amount,
            participant_state.nonce,
            participant_state.locksroot
);
```
Would be nice if one didn't need to look that up. You can maybe add it as an comment.

Reply:
I've changed it to a dict..